demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -130,9 +130,83 @@ static int on_end_stream(nghttp3_conn *h3conn, int64_t stream_id,
     return 0;
 }
 
-static int read_from_ssl_ids(nghttp3_conn *h3conn, struct h3ssl *h3ssl)
+/* Read from the stream and push to the h3conn */
+static int quic_server_read(nghttp3_conn *h3conn, SSL *stream, uint64_t id)
 {
+    int ret, r;
     uint8_t msg2[16000];
+    size_t l = sizeof(msg2) - 1;
+
+    if (!SSL_net_read_desired(stream))
+        return 0; /* Nothing to read */
+
+    ret = SSL_read(stream, msg2, l);
+    if (ret <= 0) {
+        fprintf(stderr, "SSL_read on %llu failed\n",
+               (unsigned long long) id);
+        return -1;
+    }
+
+    r = nghttp3_conn_read_stream(h3conn, id, msg2, ret, 0);
+
+    printf("reading something %d on %llu\n", ret,
+           (unsigned long long) id);
+    printf("nghttp3_conn_read_stream used %d of %d on %llu\n", r,
```

Comment:
Are these debugging printfs?  Should they be removed?

Reply:
Yes those are debug printf() I have removed the first one that is useless.